src/SWP/Bundle/AnalyticsBundle/Model/RequestMetric.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\AnalyticsBundle\Model;
+
+/**
+ * RequestMetric -.
```

Comment:
this comment can be removed if its the same a class name, it doesnt tell much anyway